rust/connlib/clients/android/src/lib.rs

```diff
@@ -340,12 +357,19 @@ fn connect(
                     source,
                 })?,
         );
+
+    let log_string = String::from(env.get_string(&log_string).map_err(|source| {
+        ConnectError::StringInvalid {
+            name: "log_string",
+            source,
+        }
+    })?);
```

Comment:
We should be able to extract a utility function here that avoids the duplication of getting a string from `env`.